clickhouse/src/datasources/click-house-datasource/ClickHouseDatasourceEditor.tsx

```diff
@@ -0,0 +1,90 @@
+import { OptionsEditorColumn, OptionsEditorControl, OptionsEditorGrid } from '@perses-dev/components';
```

Comment:
Can you add the license header also here and in every typescript missing it ? Thanks